packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -514,6 +514,10 @@ class _InteractiveViewerState extends State<InteractiveViewer> with TickerProvid
     final Rect boundaryRect = widget.boundaryMargin.inflateRect(Offset.zero & childSize);
     // Boundaries that are partially infinite are not allowed because Matrix4's
     // rotation and translation methods don't handle infinites well.
+    assert(
```

Comment:
nit: the comment above is probably for the next assert.

Reply:
Thanks, good catch.